service/routes.py

```diff
@@ -125,13 +125,31 @@ def delete_promotion(promotion_id):
     app.logger.info("Request to delete a promotion with id %s", promotion_id)
     promo = Promotion.find(promotion_id)
     if not promo:
-        return (
-            "Did not find promotions with given ID",
-            status.HTTP_204_NO_CONTENT
-        )
+        abort(status.HTTP_404_NOT_FOUND, f'Promotion with id {promotion_id} was not found.')
```

Comment:
I just update this part to return 204 when not found the corresponding promotions. In last sprint this part return 404, but prof said delete endpoint should always return 204_NO_CONTENT

Reply:
since this is updated, the test case for delete_not_found also needs to be updated

Reply:
@tanaytrivedi this needs to be `HTTP_204_NO_CONTENT`